### PR TITLE
Fixes #24526 - handle compute resource fingerprint error

### PR DESCRIPTION
**Pull request by iNecas**

### CODE REVIEW

**Comment by ares** [comment]
Tests failures seem related

**Comment by iNecas** [comment]
I needed to include the helper outside of the test module, as it was causing some `test_methods` to be running with assumption that they are actual tests. The Jenkins should turn green soon. Ready for re-review.

File: test/helpers/compute_resources_helper_test.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+require 'test_helper'
+
+class ComputeResourcesHelperTest < ActionView::TestCase
+  class DummyController
+    include ComputeResourcesHelper
+
+    delegate :controller, to: :@context
```
Comment (ares) [inline test/helpers/compute_resources_helper_test.rb:7]: I didn't know this works :-)

**Review by ares — APPROVED** [review]
audits issue on sqlite seems unrelated, merging, thanks @iNecas

**Comment by tbrisker** [comment]
Cherry-pick to 1.19-stable as 8de1c8220

**Pull request merged**